source/ConditionsStore.h

```diff
@@ -37,7 +38,12 @@ class DataWriter;
 // data types than int64_t (for example double, float or even complex
 // formulae).
 class ConditionsStore {
```

Comment:
```suggestion
class ConditionsStore {
private:
```

Reply:
Responses:

1. It wouldn't match what is done in the other classes in ConditionsStore.
2. A friend block has the same meaning whether it is in a public block or private block.
3. The default for classes is private anyway.

Reply:
Uh, there has to be something wrong with the other conditionstore class, cause that's how e do it.

Reply:
I'm keeping the declarations consistent with the style of the rest of the file that has those declarations. If you want the style to change, you need to talk to @petervdmeer who wrote that file.